demo/src/screens/ProductScreen/ProductScreen.js

```diff
@@ -1,13 +1,100 @@
 import React from 'react';
+import {Container, Row, Col, Image, Button} from 'react-bootstrap';
+import {useParams} from 'react-router-dom';
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+import {Link} from 'react-router-dom';
+import {addOneToCart} from '../../store/StoreHelpers.js';
+
 
 /**
  * @return {!JSX} Page component for where a user can view
  *     details about a single product.
  */
-export function ProductScreen() {
+const ProductScreenBase = function({items, addToCart}) {
```

Comment:
Can you please add a JSDoc annotation for the parameter here?

Reply:
Done!